contracts/Polygon_SpokePool.sol

```diff
@@ -188,6 +243,18 @@ contract Polygon_SpokePool is IFxMessageProcessor, SpokePool {
         if (balance > 0) wrappedNativeToken.deposit{ value: balance }();
     }
 
+    function _setFunctionLock(bytes32 funcSig) internal {
```

Comment:
nit: since we're just using custom string identifiers, we might want to rename this arg (here and in the other function) to `funcIdentifier` or similar, since a function "signature" has a specific meaning.